recipe/meta.yaml

```diff
@@ -25,22 +24,23 @@ requirements:
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
     - numpy {{ numpy }}
     - cython
     - packaging
-
+    - setuptools
   run:
     - python
     - scipy >=1.5.0
-    - {{ pin_compatible('numpy', lower_bound='1.22.3') }}
+    - numpy >=1.23.2
     - matplotlib-base >=1.5.1
     - seaborn    # [not ppc64le]
     - netcdf4 >=1.0
-    - biopython >=1.80
+    - biopython >=1.80  # [not py>312]
```

Comment:
Does biopython not work for 3.13 yet?

Reply:
Not as far as I can tell, their migrator is currently up.